pkgdb2/templates/list_actions.html

```diff
@@ -58,7 +58,7 @@
     <tr>
         <td>
         {% if page > 1%}
-            <a href="{{ url_for('.admin_action') }}?page={{page - 1}}&package={{package}}&from_date={{from_date}}">
+            <a href="{{ url_for('.admin_actions') }}?page={{page - 1}}&package={{package}}&from_date={{from_date}}">
```

Comment:
While we are at fixing this template, could you add the page, package and from_date directly in the ``url_for()``

This would avoid the pattern ``?package=&from_date=&page=2`` as it will automatically drop the empty arguments